src/components/VoiceRecorder/VoiceRecorder.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Mic from '@material-ui/icons/Mic';
+import './VoiceRecorder.css';
+
+export default class VoiceRecorder extends React.Component {
+  state = {
+    record: false,
+    mediaRecorder: '',
```

Comment:
We want to keep the `state` object as simple as it can be. Remember that `state` changes re-render the component and we want to avoid re-rendering when nothing changes *visually*. So I think we can move `record`, `mediaRecorder` and `chunks` from the `state` object to the class instance itself. The same way you defined `state = {}` on the class, we should move the properties to the class.